### PR TITLE
Task Card Completions Working as Intended

### PR DESCRIPTION
**Pull request by lavanv11**

### CODE REVIEW

**Comment by lavanv11** [comment]
Note: The issue on simulator -> Double Completions -> Has not been fixed as it has only been observed to occur in simulator.

**Review by roshankern — APPROVED** [review]
LGTM!

File: flutter_01.png
Comment (roshankern) [inline flutter_01.png:1]: Actually just noticed this image is being added to the repo. We probably dont need this right? lol

Comment (lavanv11) [reply flutter_01.png:1]: Yeah we don't or at least we shouldn't.

**Pull request merged**